packages/medusa/src/services/draft-order.js

```diff
@@ -249,8 +249,13 @@ class DraftOrderService extends BaseService {
         )
       }
 
-      const { items, shipping_methods, discounts, no_notification_order=undefined, ...rest } = data
-
+      const {
+        items,
+        shipping_methods,
+        discounts,
+        no_notification_order = undefined,
```

Comment:
Don't do `= undefined` as `no_notification_order` would already be undefined if not set. If you want to be explicit about it add a comment when you evaluate the value.